pims/display.py

```diff
@@ -387,20 +387,26 @@ def _fig_size_cntx(fig, fig_size_inches):
     fig : matplotlib.figure.Figure
         The figure to resize
     fig_size_inches : tuple
-        The (height, width) to use in the context.  If None, the size
+        The (height, width) to use in the context. If None, the size
         is not changed
-
+    tight_layout : boolean or None
+        When True, tight layout is used.
     """
     orig_size = fig.get_size_inches()
+    orig_layout = fig.get_tight_layout()
     if fig_size_inches is not None:
         fig.set_size_inches(*fig_size_inches)
+    if tight_layout is not None:
+        fig.set_tight_layout(tight_layout)
     try:
         yield fig
     finally:
         fig.set_size_inches(*orig_size)
+        fig.set_tight_layout(orig_layout)
 
 
-def plot_to_frame(fig, width=512, close_fig=False, bbox_inches=None):
+def plot_to_frame(fig, width=512, close_fig=False, fig_size_inches=None,
```

Comment:
There was already an API break in #204; in the latest release (v0.3.2) it has the following arguments: `fig, dpi, **imsave_kwargs`. I will document that.